gr-iio/lib/iio_priv.h

```diff
@@ -0,0 +1,29 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2023 Analog Devices Inc.
+ * Author: Adrian Suciu <adrian.suciu@analog.com>
+
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifndef IIO_PRIV_H
```

Comment:
To stay consistent with the other headers in this directory (and to avoid conflict with any IIO headers of the future?), I'd go with `INCLUDED_IIO_IIO_PRIV_H` instead

Reply:
Fixed